lib/floe/workflow/runner.rb

```diff
@@ -5,11 +5,14 @@ class Workflow
     class Runner
       include Logging
 
+      def initialize(_options = {})
+      end
+
       class << self
-        attr_writer :docker_runner_klass
+        attr_writer :docker_runner
```

Comment:
Wonder if this should just be `runner` (i.e. remove the docker bit) - in the future we may not have docker-specific runners - for example, built-in methods or amazon lambdas.

Reply:
ohhhh no, wait, this is specifically how to run the `docker://` scheme - this is fine.

Reply:
It was actually because we might have non-docker resources like `lambda://` that I named this docker_runner instead of just runner.  We would have a runner class for each resource type that we support.